.github/workflows/cron-ci.yaml

```diff
@@ -0,0 +1,49 @@
+# This workflow runs on a weekly basis and validates the integrity of the
+# build and links.
+
+name: Weekly CI
+
+on:
+  schedule:
+    - cron: "0 4 * * MON"
+
+jobs:
+  linkcheck:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2.1.2
+        with:
+          python-version: 3.8
+
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools
+          pip install -r requirements.txt
+
+      - name: Runk linkcheck
+        run: make linkcheck
+
+  build:
```

Comment:
I'm wondering how you decided to run in parallel rather than in sequence here.  Is it for performance or to catch problems if either job finds them or something else?

Reply:
Right, it ensures that both the link check and the builds happen regardless of the other failing. The parallelization is a bonus but doesn't matter much for the cron-based job.